From ae21cc86ab9eea0e0c927cc13decb296a42fc5f5 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Mon, 13 Jan 2020 18:21:39 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-excavator-sapphire: set dmc
 boost to 600m

dmc set SYS_STATUS_BOOST 600MHZ for lpddr3

Change-Id: I6c1905a284ec5986b737a285fb96db19d25f3c96
Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
index 98f5da90dca1..c6de919f11f8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
@@ -144,7 +144,7 @@
 		SYS_STATUS_VIDEO_4K     600000
 		SYS_STATUS_VIDEO_4K_10B 800000
 		SYS_STATUS_PERFORMANCE  800000
-		SYS_STATUS_BOOST        400000
+		SYS_STATUS_BOOST        600000
 		SYS_STATUS_DUALVIEW     600000
 		SYS_STATUS_ISP          600000
 	>;
-- 
2.35.3

